packages/terra-infinite-list/src/terra-dev-site/test/infinite-list/InfiniteListSameCount.test.scss

```diff
@@ -0,0 +1,23 @@
+@import './InfiniteListTestCommon';
+
+:local {
+  .item-type1 {
```

Comment:
nit: I'd consolidate these down to 1 class, `.item`. We don't have much need to split these into separate classes at this time.

Reply:
The reason i had to split the items into separate classes was because each item has a different background color.